editor/doc_data.h

```diff
@@ -111,8 +129,18 @@ class DocData {
 	Error _load(Ref<XMLParser> parser);
 
 public:
+	static void return_doc_from_retinfo(DocData::MethodDoc &p_method, const PropertyInfo &p_retinfo);
+	static void argument_doc_from_arginfo(DocData::ArgumentDoc &p_argument, const PropertyInfo &p_arginfo);
+	static void property_doc_from_scriptmemberinfo(DocData::PropertyDoc &p_property, const ScriptMemberInfo &p_memberinfo);
+	static void method_doc_from_methodinfo(DocData::MethodDoc &p_method, const MethodInfo &p_methodinfo);
+	static void constant_doc_from_variant(DocData::ConstantDoc &p_const, const StringName &p_name, const Variant &p_value, const String &p_desc);
+	static void signal_doc_from_methodinfo(DocData::MethodDoc &p_signal, const MethodInfo &p_methodinfo);
+
 	void merge_from(const DocData &p_data);
 	void remove_from(const DocData &p_data);
+	void add_doc(const ClassDoc &p_class_doc);
+	void remove_doc(const String &p_class_name);
+	bool has_doc(const String &p_class_name);
```

Comment:
Nitpick, but the argument names differ a bit here.

Reply:
I match the argument name with it's type add with the `ClassDoc` and remove with it's name